### DISCUSSION TITLE
can not add new alert channel with signoz V0.56

### DISCUSSION THREAD

**Discussion by victorChou1024** [SigNoz/signoz#6240] [Q&A] [answered]
I set up the signoz0.56 with docker-compose. Everything works well. but when i tested the alert feature by the ADMIN role, the SAVE button in the alert channel configuration page is gray and could not be clicked.

**Comment by prashant-shahi** [comment]
Was the user account recently bumped up to the ADMIN role?

If yes, you will need to log out and re-login.

**Comment by victorChou1024** [comment]
![signoz-alertchannel](https://github.com/user-attachments/assets/4fc99d7e-3380-416b-b9b1-8e9038441576)
Thanks for your reply.
 It didn't work after i re-login
And i am trying the Free Edition
Does it relate  to license matters?

**Comment by victorChou1024** [comment]
![0 56](https://github.com/user-attachments/assets/a8300f83-5b42-469d-855c-db68a9b02e1e)
![0 54](https://github.com/user-attachments/assets/e2685c34-e776-4473-ad66-4d0b8ec25533)
I find that when call the interface /api/v1/featureFlags at browser, the response is different between 0.56 and 0.54.
It seems that the 0.56 response miss some feature information.

**Reply by ankitnayan** [reply]
@makeavish please check

**Reply by makeavish** [reply]
@victorChou1024 Are you using a custom/different build?
Can you please install with `./install.sh` and let us know if works

**Reply by victorChou1024** [reply, marked as answer]
> @victorChou1024 Are you using a custom/different build? Can you please install with `./install.sh` and let us know if works
Thanks.
Problem solved. It came with the incompatible images. I compiled the query-serive image myself.
